src/hotspot/share/gc/g1/g1RemSet.cpp

```diff
@@ -1763,7 +1763,7 @@ void G1RemSet::enqueue_for_reprocessing(CardValue* card_ptr) {
   dcqs.enqueue_completed_buffer(BufferNode::make_node_from_buffer(buffer, index));
 }
 
-void G1RemSet::print_periodic_summary_info(const char* header, uint period_count) {
+void G1RemSet::print_periodic_summary_info(const char* header, uint period_count, bool include_vtimes) {
```

Comment:
```suggestion
void G1RemSet::print_periodic_summary_info(const char* header, uint period_count, bool show_thread_times) {
```

I think the suggested parameter name is more clear than the original one.

Reply:
Updated.